index.bs

```diff
@@ -2537,13 +2537,15 @@ Short Circuit</h5>
 	and potential members of a Council who are not renouncing their seat
 	confirm it by a vote which results in both of the following:
 	* at least 80% of them vote affirmatively to adopt this resolution
+      (in addition, Tim Berners-Lee <em class=rfc2119>may</em> also abstain)
```

Comment:
```suggestion
```
Thanks for rebasing, and I agree with you that afterwards, this reads more awkwardly than helpful.
Also in taking a step back and considering the broader "why" of this PR, I think an overlooked aspect (which may lead to a cleaner solution) is that a short-circuit is in the context of a *Team* recommendation — this implies to me that the folks already on or affiliated in some way with the Team (including TimBL, both historically and possibly in a remaining soft influence way) have already had a chance to contribute their opinions (perhaps even solicited to do so) as part of that Team recommendation, and presumably are at least "ok" with it, and thus don't need to "double-vote" as it were (once as part of crafting the recommendation, and then again as part of the short-circuit).
Thus I'd drop this line entirely (as it is awkward), and I will propose a different, much smaller change that I believe both addressing the implicit intent of this PR, as well as the broader "problem" it is solving a part of.